In [1]:
import numpy as np
import pandas as pd
import os

import tensorflow as tf
import tensorflow_decision_forests as tfdf

print(f"Found TF-DF {tfdf.__version__}")

Found TF-DF 1.9.1


In [2]:
train_df = pd.read_csv("/kaggle/input/titanic/train.csv")
serving_df = pd.read_csv("/kaggle/input/titanic/test.csv")

train_df.head(10)

PassengerId  Survived  Pclass  \
0            1         0       3   
1            2         1       1   
2            3         1       3   
3            4         1       1   
4            5         0       3   
5            6         0       3   
6            7         0       1   
7            8         0       3   
8            9         1       3   
9           10         1       2   

                                                Name     Sex   Age  SibSp  \
0                            Braund, Mr. Owen Harris    male  22.0      1   
1  Cumings, Mrs. John Bradley (Florence Briggs Th...  female  38.0      1   
2                             Heikkinen, Miss. Laina  female  26.0      0   
3       Futrelle, Mrs. Jacques Heath (Lily May Peel)  female  35.0      1   
4                           Allen, Mr. William Henry    male  35.0      0   
5                                   Moran, Mr. James    male   NaN      0   
6                            McCarthy, Mr. Timothy J    male  54.0      0   
7                     Palsson, Master. Gosta Leonard    male   2.0      3   
8  Johnson, Mrs. Oscar W (Elisabeth Vilhelmina Berg)  female  27.0      0   
9                Nasser, Mrs. Nicholas (Adele Achem)  female  14.0      1   

   Parch            Ticket     Fare Cabin Embarked  
0      0         A/5 21171   7.2500   NaN        S  
1      0          PC 17599  71.2833   C85        C  
2      0  STON/O2. 3101282   7.9250   NaN        S  
3      0            113803  53.1000  C123        S  
4      0            373450   8.0500   NaN        S  
5      0            330877   8.4583   NaN        Q  
6      0             17463  51.8625   E46        S  
7      1            349909  21.0750   NaN        S  
8      2            347742  11.1333   NaN        S  
9      0            237736  30.0708   NaN        C

Tokenize the names. For example, "Braund, Mr. Owen Harris" will become ["Braund", "Mr.", "Owen", "Harris"].

Extract any prefix in the ticket. For example ticket "STON/O2. 3101282" will become "STON/O2." and 3101282.

In [3]:
def preprocess(df):
    df = df.copy()
    
    def normalize_name(x):
        return " ".join([v.strip(",()[].\"'") for v in x.split(" ")])
    
    def ticket_number(x):
        return x.split(" ")[-1]
        
    def ticket_item(x):
        items = x.split(" ")
        if len(items) == 1:
            return "NONE"
        return "_".join(items[0:-1])
    
    df["Name"] = df["Name"].apply(normalize_name)
    df["Ticket_number"] = df["Ticket"].apply(ticket_number)
    df["Ticket_item"] = df["Ticket"].apply(ticket_item)                     
    return df
    
preprocessed_train_df = preprocess(train_df)
preprocessed_serving_df = preprocess(serving_df)

preprocessed_train_df.head(5)

PassengerId  Survived  Pclass  \
0            1         0       3   
1            2         1       1   
2            3         1       3   
3            4         1       1   
4            5         0       3   

                                              Name     Sex   Age  SibSp  \
0                            Braund Mr Owen Harris    male  22.0      1   
1  Cumings Mrs John Bradley Florence Briggs Thayer  female  38.0      1   
2                             Heikkinen Miss Laina  female  26.0      0   
3         Futrelle Mrs Jacques Heath Lily May Peel  female  35.0      1   
4                           Allen Mr William Henry    male  35.0      0   

   Parch            Ticket     Fare Cabin Embarked Ticket_number Ticket_item  
0      0         A/5 21171   7.2500   NaN        S         21171         A/5  
1      0          PC 17599  71.2833   C85        C         17599          PC  
2      0  STON/O2. 3101282   7.9250   NaN        S       3101282    STON/O2.  
3      0            113803  53.1000  C123        S        113803        NONE  
4      0            373450   8.0500   NaN        S        373450        NONE

Let's keep the list of the input features of the model. Notably, we don't want to train our model on the "PassengerId" and "Ticket" features.

In [4]:
input_features = list(preprocessed_train_df.columns)
input_features.remove("Ticket")
input_features.remove("PassengerId")
input_features.remove("Survived")
#input_features.remove("Ticket_number")

print(f"Input features: {input_features}")

Input features: ['Pclass', 'Name', 'Sex', 'Age', 'SibSp', 'Parch', 'Fare', 'Cabin', 'Embarked', 'Ticket_number', 'Ticket_item']


In [5]:
#Convert Pandas dataset to TensorFlow Dataset
def tokenize_names(features, labels=None):
    """Divite the names into tokens. TF-DF can consume text tokens natively."""
    features["Name"] =  tf.strings.split(features["Name"])
    return features, labels

train_ds = tfdf.keras.pd_dataframe_to_tf_dataset(preprocessed_train_df,label="Survived").map(tokenize_names)
serving_ds = tfdf.keras.pd_dataframe_to_tf_dataset(preprocessed_serving_df).map(tokenize_names)

Train model with default parameters¶


In [6]:
#First, we are training a GradientBoostedTreesModel model with the default parameters.

model = tfdf.keras.GradientBoostedTreesModel(
    verbose=0, # Very few logs
    features=[tfdf.keras.FeatureUsage(name=n) for n in input_features],
    exclude_non_specified_features=True, # Only use the features in "features"
    random_seed=1234,
)
model.fit(train_ds)

self_evaluation = model.make_inspector().evaluation()
print(f"Accuracy: {self_evaluation.accuracy} Loss:{self_evaluation.loss}")

[WARNING 24-09-03 16:46:19.9120 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:46:19.9128 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:46:19.9128 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:46:26.4316 UTC kernel.cc:1233] Loading model from path /tmp/tmpw476adgl/model/ with prefix 8742c209210e43a5
[INFO 24-09-03 16:46:26.4370 UTC quick_scorer_extended.cc:911] The binary was compiled without AVX2 support, but your CPU supports it. Enable it for faster model inference.
[INFO 24-09-03 16:46:26.4375 UTC abstract_model.cc:1362] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 24-09-03 16:46:26.4375 UTC kernel.cc:1061] Use fast generic engine


Accuracy: 0.8260869383811951 Loss:0.8608942627906799


Train model with improved default parameters¶
Now you'll use some specific parameters when creating the GBT model

In [7]:
model = tfdf.keras.GradientBoostedTreesModel(
    verbose=0, # Very few logs
    features=[tfdf.keras.FeatureUsage(name=n) for n in input_features],
    exclude_non_specified_features=True, # Only use the features in "features"
    
    #num_trees=2000,
    
    # Only for GBT.
    # A bit slower, but great to understand the model.
    # compute_permutation_variable_importance=True,
    
    # Change the default hyper-parameters
    # hyperparameter_template="benchmark_rank1@v1",
    
    #num_trees=1000,
    #tuner=tuner
    
    min_examples=1,
    categorical_algorithm="RANDOM",
    #max_depth=4,
    shrinkage=0.05,
    #num_candidate_attributes_ratio=0.2,
    split_axis="SPARSE_OBLIQUE",
    sparse_oblique_normalization="MIN_MAX",
    sparse_oblique_num_projections_exponent=2.0,
    num_trees=2000,
    #validation_ratio=0.0,
    random_seed=1234,
    
)
model.fit(train_ds)

self_evaluation = model.make_inspector().evaluation()
print(f"Accuracy: {self_evaluation.accuracy} Loss:{self_evaluation.loss}")

[WARNING 24-09-03 16:46:36.0363 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:46:36.0364 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:46:36.0364 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:46:36.7873 UTC kernel.cc:1233] Loading model from path /tmp/tmp51hlm8fi/model/ with prefix 47348cf6f44c4ea5
[INFO 24-09-03 16:46:36.7952 UTC decision_forest.cc:734] Model loaded with 40 root(s), 2106 node(s), and 10 input feature(s).
[INFO 24-09-03 16:46:36.7952 UTC abstract_model.cc:1362] Engine "GradientBoostedTreesGeneric" built
[INFO 24-09-03 16:46:36.7953 UTC kernel.cc:1061] Use fast generic engine


Accuracy: 0.782608687877655 Loss:1.0586705207824707


In [8]:
model.summary()

Model: "gradient_boosted_trees_model_1"
_________________________________________________________________
 Layer (type)                Output Shape              Param #   
Total params: 1 (1.00 Byte)
Trainable params: 0 (0.00 Byte)
Non-trainable params: 1 (1.00 Byte)
_________________________________________________________________
Type: "GRADIENT_BOOSTED_TREES"
Task: CLASSIFICATION
Label: "__LABEL"

Input Features (11):
	Age
	Cabin
	Embarked
	Fare
	Name
	Parch
	Pclass
	Sex
	SibSp
	Ticket_item
	Ticket_number

No weights

Variable Importance: INV_MEAN_MIN_DEPTH:
    1.           "Sex"  0.585997 ################
    2.           "Age"  0.364636 #######
    3.          "Fare"  0.266191 ###
    4.          "Name"  0.207054 #
    5.        "Pclass"  0.179191 
    6. "Ticket_number"  0.178806 
    7.      "Embarked"  0.177803 
    8.   "Ticket_item"  0.177009 
    9.         "Parch"  0.175276 
   10.         "SibSp"  0.171694 

Variable Importance: NUM_AS_ROOT:
    1.  "Sex" 34.000000 ######

Make predictions¶


In [9]:
def prediction_to_kaggle_format(model, threshold=0.5):
    proba_survive = model.predict(serving_ds, verbose=0)[:,0]
    return pd.DataFrame({
        "PassengerId": serving_df["PassengerId"],
        "Survived": (proba_survive >= threshold).astype(int)
    })

def make_submission(kaggle_predictions):
    path="/kaggle/working/submission.csv"
    kaggle_predictions.to_csv(path, index=False)
    print(f"Submission exported to {path}")
    
kaggle_predictions = prediction_to_kaggle_format(model)
make_submission(kaggle_predictions)
!head /kaggle/working/submission.csv

Submission exported to /kaggle/working/submission.csv


/opt/conda/lib/python3.10/pty.py:89: RuntimeWarning: os.fork() was called. os.fork() is incompatible with multithreaded code, and JAX is multithreaded, so this will likely lead to a deadlock.
  pid, fd = os.forkpty()


PassengerId,Survived
892,0
893,0
894,0
895,0
896,0
897,0
898,0
899,0
900,1


Training a model with hyperparameter tunning¶

Hyper-parameter tuning is enabled by specifying the tuner constructor argument of the model. The tuner object contains all the configuration of the tuner (search space, optimizer, trial and objective).

In [10]:
tuner = tfdf.tuner.RandomSearch(num_trials=1000)
tuner.choice("min_examples", [2, 5, 7, 10])
tuner.choice("categorical_algorithm", ["CART", "RANDOM"])

local_search_space = tuner.choice("growing_strategy", ["LOCAL"])
local_search_space.choice("max_depth", [3, 4, 5, 6, 8])

global_search_space = tuner.choice("growing_strategy", ["BEST_FIRST_GLOBAL"], merge=True)
global_search_space.choice("max_num_nodes", [16, 32, 64, 128, 256])

#tuner.choice("use_hessian_gain", [True, False])
tuner.choice("shrinkage", [0.02, 0.05, 0.10, 0.15])
tuner.choice("num_candidate_attributes_ratio", [0.2, 0.5, 0.9, 1.0])


tuner.choice("split_axis", ["AXIS_ALIGNED"])
oblique_space = tuner.choice("split_axis", ["SPARSE_OBLIQUE"], merge=True)
oblique_space.choice("sparse_oblique_normalization",
                     ["NONE", "STANDARD_DEVIATION", "MIN_MAX"])
oblique_space.choice("sparse_oblique_weights", ["BINARY", "CONTINUOUS"])
oblique_space.choice("sparse_oblique_num_projections_exponent", [1.0, 1.5])

# Tune the model. Notice the `tuner=tuner`.
tuned_model = tfdf.keras.GradientBoostedTreesModel(tuner=tuner)
tuned_model.fit(train_ds, verbose=0)

tuned_self_evaluation = tuned_model.make_inspector().evaluation()
print(f"Accuracy: {tuned_self_evaluation.accuracy} Loss:{tuned_self_evaluation.loss}")

Use /tmp/tmphk4jfu10 as temporary training directory


[WARNING 24-09-03 16:46:38.4149 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:46:38.4150 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:46:38.4150 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:51:33.6119 UTC kernel.cc:1233] Loading model from path /tmp/tmphk4jfu10/model/ with prefix 81e0b637a86447c0
[INFO 24-09-03 16:51:33.6266 UTC decision_forest.cc:734] Model loaded with 30 root(s), 2236 node(s), and 12 input feature(s).
[INFO 24-09-03 16:51:33.6266 UTC abstract_model.cc:1362] Engine "GradientBoostedTreesGeneric" built
[INFO 24-09-03 16:51:33.6266 UTC kernel.cc:1061] Use fast generic engine


Accuracy: 0.8630136847496033 Loss:0.6749962568283081


Making an ensemble
Here you'll create 100 models with different seeds and combine their results

This approach removes a little bit the random aspects related to creating ML models

In the GBT creation is used the honest parameter. It will use different training examples to infer the structure and the leaf values. This regularization technique trades examples for bias estimates.

In [11]:
predictions = None
num_predictions = 0

for i in range(100):
    print(f"i:{i}")
    # Possible models: GradientBoostedTreesModel or RandomForestModel
    model = tfdf.keras.GradientBoostedTreesModel(
        verbose=0, # Very few logs
        features=[tfdf.keras.FeatureUsage(name=n) for n in input_features],
        exclude_non_specified_features=True, # Only use the features in "features"

        #min_examples=1,
        #categorical_algorithm="RANDOM",
        ##max_depth=4,
        #shrinkage=0.05,
        ##num_candidate_attributes_ratio=0.2,
        #split_axis="SPARSE_OBLIQUE",
        #sparse_oblique_normalization="MIN_MAX",
        #sparse_oblique_num_projections_exponent=2.0,
        #num_trees=2000,
        ##validation_ratio=0.0,
        random_seed=i,
        honest=True,
    )
    model.fit(train_ds)
    
    sub_predictions = model.predict(serving_ds, verbose=0)[:,0]
    if predictions is None:
        predictions = sub_predictions
    else:
        predictions += sub_predictions
    num_predictions += 1

predictions/=num_predictions

kaggle_predictions = pd.DataFrame({
        "PassengerId": serving_df["PassengerId"],
        "Survived": (predictions >= 0.5).astype(int)
    })

make_submission(kaggle_predictions)

i:0


[WARNING 24-09-03 16:51:34.0054 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:51:34.0054 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:51:34.0054 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:51:34.4385 UTC kernel.cc:1233] Loading model from path /tmp/tmp1jvlcid0/model/ with prefix 7c4ea98d7dc04c47
[INFO 24-09-03 16:51:34.4422 UTC quick_scorer_extended.cc:911] The binary was compiled without AVX2 support, but your CPU supports it. Enable it for faster model inference.
[INFO 24-09-03 16:51:34.4426 UTC kernel.cc:1061] Use fast generic engine


i:1


[WARNING 24-09-03 16:51:34.8397 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:51:34.8397 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:51:34.8397 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:51:35.5832 UTC kernel.cc:1233] Loading model from path /tmp/tmpkr8bjlc3/model/ with prefix e00d0ebd93c54b88
[INFO 24-09-03 16:51:35.6006 UTC kernel.cc:1061] Use fast generic engine


i:2


[WARNING 24-09-03 16:51:36.0038 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:51:36.0039 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:51:36.0039 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:51:36.4438 UTC kernel.cc:1233] Loading model from path /tmp/tmp6v1lysn1/model/ with prefix 47c555123b494a32
[INFO 24-09-03 16:51:36.4478 UTC kernel.cc:1061] Use fast generic engine


i:3


[WARNING 24-09-03 16:51:36.8502 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:51:36.8503 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:51:36.8503 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:51:37.7219 UTC kernel.cc:1233] Loading model from path /tmp/tmpwj_vnsp8/model/ with prefix fed883768fb44d0f
[INFO 24-09-03 16:51:37.7461 UTC kernel.cc:1061] Use fast generic engine


i:4


[WARNING 24-09-03 16:51:38.1600 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:51:38.1601 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:51:38.1601 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:51:38.6243 UTC kernel.cc:1233] Loading model from path /tmp/tmpf000assk/model/ with prefix 3cc022da2d354761
[INFO 24-09-03 16:51:38.6294 UTC kernel.cc:1061] Use fast generic engine


i:5


[WARNING 24-09-03 16:51:39.0380 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:51:39.0380 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:51:39.0381 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:51:39.4526 UTC kernel.cc:1233] Loading model from path /tmp/tmpocfqcoa1/model/ with prefix f4a91452994547c4
[INFO 24-09-03 16:51:39.4553 UTC kernel.cc:1061] Use fast generic engine


i:6


[WARNING 24-09-03 16:51:39.8819 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:51:39.8820 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:51:39.8820 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:51:40.3956 UTC kernel.cc:1233] Loading model from path /tmp/tmpryh06db7/model/ with prefix 009d5f390f924cdf
[INFO 24-09-03 16:51:40.4025 UTC kernel.cc:1061] Use fast generic engine


i:7


[WARNING 24-09-03 16:51:40.8205 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:51:40.8206 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:51:40.8206 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:51:41.5962 UTC kernel.cc:1233] Loading model from path /tmp/tmpxwh0l_cr/model/ with prefix 59892d23be664107
[INFO 24-09-03 16:51:41.6147 UTC kernel.cc:1061] Use fast generic engine


i:8


[WARNING 24-09-03 16:51:42.0388 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:51:42.0388 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:51:42.0388 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:51:42.5971 UTC kernel.cc:1233] Loading model from path /tmp/tmpw9qj6wlk/model/ with prefix d915f806b13c4702
[INFO 24-09-03 16:51:42.6058 UTC kernel.cc:1061] Use fast generic engine


i:9


[WARNING 24-09-03 16:51:43.0287 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:51:43.0287 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:51:43.0287 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:51:43.6890 UTC kernel.cc:1233] Loading model from path /tmp/tmpnlyli2y9/model/ with prefix 49b56efc8f0e4483
[INFO 24-09-03 16:51:43.7022 UTC abstract_model.cc:1362] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 24-09-03 16:51:43.7022 UTC kernel.cc:1061] Use fast generic engine


i:10


[WARNING 24-09-03 16:51:44.1254 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:51:44.1255 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:51:44.1255 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:51:44.6032 UTC kernel.cc:1233] Loading model from path /tmp/tmpn1bgfds0/model/ with prefix 539034c9932e4c5f
[INFO 24-09-03 16:51:44.6083 UTC kernel.cc:1061] Use fast generic engine


i:11


[WARNING 24-09-03 16:51:45.0264 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:51:45.0264 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:51:45.0264 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:51:45.6729 UTC kernel.cc:1233] Loading model from path /tmp/tmpu0exz_fb/model/ with prefix d9ff2b63ff2e44a3
[INFO 24-09-03 16:51:45.6859 UTC kernel.cc:1061] Use fast generic engine


i:12


[WARNING 24-09-03 16:51:46.0938 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:51:46.0938 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:51:46.0938 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:51:46.5847 UTC kernel.cc:1233] Loading model from path /tmp/tmp3ip10sh5/model/ with prefix 619221b7375246b6
[INFO 24-09-03 16:51:46.5900 UTC kernel.cc:1061] Use fast generic engine


i:13


[WARNING 24-09-03 16:51:46.9999 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:51:46.9999 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:51:46.9999 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:51:47.5837 UTC kernel.cc:1233] Loading model from path /tmp/tmpp6vjr3lw/model/ with prefix ab0c48d4caee4f2b
[INFO 24-09-03 16:51:47.5936 UTC kernel.cc:1061] Use fast generic engine


i:14


[WARNING 24-09-03 16:51:48.0025 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:51:48.0026 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:51:48.0026 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:51:48.4799 UTC kernel.cc:1233] Loading model from path /tmp/tmp5upjwmuw/model/ with prefix 6d4d809b0268441f
[INFO 24-09-03 16:51:48.4856 UTC kernel.cc:1061] Use fast generic engine


i:15


[WARNING 24-09-03 16:51:48.8929 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:51:48.8929 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:51:48.8929 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:51:49.4082 UTC kernel.cc:1233] Loading model from path /tmp/tmpe_v944c5/model/ with prefix e02e04611e1e4038
[INFO 24-09-03 16:51:49.4150 UTC kernel.cc:1061] Use fast generic engine


i:16


[WARNING 24-09-03 16:51:49.8265 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:51:49.8266 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:51:49.8266 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:51:50.4648 UTC kernel.cc:1233] Loading model from path /tmp/tmp8fqju80o/model/ with prefix 5689d2b3b547489c
[INFO 24-09-03 16:51:50.4763 UTC kernel.cc:1061] Use fast generic engine


i:17


[WARNING 24-09-03 16:51:50.8839 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:51:50.8839 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:51:50.8839 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:51:51.5093 UTC kernel.cc:1233] Loading model from path /tmp/tmpkm5y18jk/model/ with prefix 9a79d3827e2f498c
[INFO 24-09-03 16:51:51.5211 UTC kernel.cc:1061] Use fast generic engine


i:18


[WARNING 24-09-03 16:51:55.5977 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:51:55.5977 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:51:55.5978 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:51:56.2240 UTC kernel.cc:1233] Loading model from path /tmp/tmpjrn6cpbd/model/ with prefix 4f1017c8922b458e
[INFO 24-09-03 16:51:56.2359 UTC abstract_model.cc:1362] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 24-09-03 16:51:56.2360 UTC kernel.cc:1061] Use fast generic engine


i:19


[WARNING 24-09-03 16:51:56.6708 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:51:56.6708 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:51:56.6708 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:51:57.4236 UTC kernel.cc:1233] Loading model from path /tmp/tmpvktafnqv/model/ with prefix f3ba83741e8a4219
[INFO 24-09-03 16:51:57.4407 UTC kernel.cc:1061] Use fast generic engine


i:20


[WARNING 24-09-03 16:51:57.8710 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:51:57.8711 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:51:57.8711 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:51:58.5648 UTC kernel.cc:1233] Loading model from path /tmp/tmp0j7e1en0/model/ with prefix 110396c6dc53421d
[INFO 24-09-03 16:51:58.5791 UTC kernel.cc:1061] Use fast generic engine


i:21


[WARNING 24-09-03 16:51:59.0078 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:51:59.0078 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:51:59.0079 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:51:59.4846 UTC kernel.cc:1233] Loading model from path /tmp/tmpjr2jcjos/model/ with prefix a6c2d5c546514481
[INFO 24-09-03 16:51:59.4895 UTC kernel.cc:1061] Use fast generic engine


i:22


[WARNING 24-09-03 16:51:59.9397 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:51:59.9398 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:51:59.9398 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:52:00.4118 UTC kernel.cc:1233] Loading model from path /tmp/tmpvoqjv074/model/ with prefix 02fe4d679ffd4fca
[INFO 24-09-03 16:52:00.4172 UTC kernel.cc:1061] Use fast generic engine


i:23


[WARNING 24-09-03 16:52:00.8211 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:00.8211 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:00.8211 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:52:01.3621 UTC kernel.cc:1233] Loading model from path /tmp/tmpi0u2wsdh/model/ with prefix c743fc8fbc434473
[INFO 24-09-03 16:52:01.3700 UTC kernel.cc:1061] Use fast generic engine


i:24


[WARNING 24-09-03 16:52:01.7766 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:01.7766 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:01.7766 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:52:02.2601 UTC kernel.cc:1233] Loading model from path /tmp/tmpczi0v9u_/model/ with prefix 7c3f8e9591dc4d49
[INFO 24-09-03 16:52:02.2654 UTC kernel.cc:1061] Use fast generic engine


i:25


[WARNING 24-09-03 16:52:02.6759 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:02.6759 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:02.6759 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:52:03.2767 UTC kernel.cc:1233] Loading model from path /tmp/tmp_90thgme/model/ with prefix ff17ee94bddd4d0b
[INFO 24-09-03 16:52:03.2871 UTC kernel.cc:1061] Use fast generic engine


i:26


[WARNING 24-09-03 16:52:03.6968 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:03.6968 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:03.6968 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:52:04.2676 UTC kernel.cc:1233] Loading model from path /tmp/tmp199me82b/model/ with prefix db3fe0bc3b464d1f
[INFO 24-09-03 16:52:04.2768 UTC kernel.cc:1061] Use fast generic engine


i:27


[WARNING 24-09-03 16:52:04.6843 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:04.6844 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:04.6844 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:52:05.1747 UTC kernel.cc:1233] Loading model from path /tmp/tmpafu43_ce/model/ with prefix 1ff1029fdb36444d
[INFO 24-09-03 16:52:05.1803 UTC quick_scorer_extended.cc:911] The binary was compiled without AVX2 support, but your CPU supports it. Enable it for faster model inference.
[INFO 24-09-03 16:52:05.1807 UTC kernel.cc:1061] Use fast generic engine


i:28


[WARNING 24-09-03 16:52:05.5900 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:05.5900 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:05.5900 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:52:06.0707 UTC kernel.cc:1233] Loading model from path /tmp/tmpk4_u0wpf/model/ with prefix d2b9d9193a5a4b1c
[INFO 24-09-03 16:52:06.0753 UTC kernel.cc:1061] Use fast generic engine


i:29


[WARNING 24-09-03 16:52:06.4763 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:06.4763 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:06.4764 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:52:07.1118 UTC kernel.cc:1233] Loading model from path /tmp/tmp0zlyknz3/model/ with prefix 13c9ad8a580148ad
[INFO 24-09-03 16:52:07.1241 UTC abstract_model.cc:1362] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 24-09-03 16:52:07.1241 UTC kernel.cc:1061] Use fast generic engine


i:30


[WARNING 24-09-03 16:52:07.5311 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:07.5312 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:07.5312 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:52:08.4298 UTC kernel.cc:1233] Loading model from path /tmp/tmpjx9leho7/model/ with prefix d3e39c4e00f148f1
[INFO 24-09-03 16:52:08.4543 UTC kernel.cc:1061] Use fast generic engine


i:31


[WARNING 24-09-03 16:52:08.8607 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:08.8607 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:08.8607 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:52:09.4106 UTC kernel.cc:1233] Loading model from path /tmp/tmptmez77vl/model/ with prefix c6df8ee502954664
[INFO 24-09-03 16:52:09.4190 UTC kernel.cc:1061] Use fast generic engine


i:32


[WARNING 24-09-03 16:52:09.8286 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:09.8286 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:09.8286 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:52:10.3232 UTC kernel.cc:1233] Loading model from path /tmp/tmptlsp4yz1/model/ with prefix 210d6c5bc2eb4cd6
[INFO 24-09-03 16:52:10.3284 UTC kernel.cc:1061] Use fast generic engine


i:33


[WARNING 24-09-03 16:52:10.7388 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:10.7388 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:10.7389 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:52:11.3824 UTC kernel.cc:1233] Loading model from path /tmp/tmpt6g6dcwy/model/ with prefix 6ab27fb8ec924263
[INFO 24-09-03 16:52:11.3940 UTC kernel.cc:1061] Use fast generic engine


i:34


[WARNING 24-09-03 16:52:11.8374 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:11.8374 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:11.8374 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:52:12.3861 UTC kernel.cc:1233] Loading model from path /tmp/tmpvv1f53lp/model/ with prefix 512478c093df4700
[INFO 24-09-03 16:52:12.3939 UTC kernel.cc:1061] Use fast generic engine


i:35


[WARNING 24-09-03 16:52:12.8111 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:12.8111 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:12.8111 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:52:13.3392 UTC kernel.cc:1233] Loading model from path /tmp/tmpoo3nied1/model/ with prefix 3a7e07e983fe4af9
[INFO 24-09-03 16:52:13.3462 UTC kernel.cc:1061] Use fast generic engine


i:36


[WARNING 24-09-03 16:52:13.7640 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:13.7640 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:13.7640 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:52:14.4387 UTC kernel.cc:1233] Loading model from path /tmp/tmpi86p00f9/model/ with prefix 301202f93b394429
[INFO 24-09-03 16:52:14.4516 UTC kernel.cc:1061] Use fast generic engine


i:37


[WARNING 24-09-03 16:52:14.8722 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:14.8723 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:14.8723 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:52:15.4277 UTC kernel.cc:1233] Loading model from path /tmp/tmpamsflwr3/model/ with prefix 00c1ade256324f22
[INFO 24-09-03 16:52:15.4356 UTC kernel.cc:1061] Use fast generic engine


i:38


[WARNING 24-09-03 16:52:15.8457 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:15.8457 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:15.8457 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:52:16.4792 UTC kernel.cc:1233] Loading model from path /tmp/tmpsxp3es3k/model/ with prefix bd8e9e8023ca4561
[INFO 24-09-03 16:52:16.4911 UTC kernel.cc:1061] Use fast generic engine


i:39


[WARNING 24-09-03 16:52:16.8951 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:16.8951 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:16.8951 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:52:17.5328 UTC kernel.cc:1233] Loading model from path /tmp/tmp3mjizgq6/model/ with prefix c4c694a790b0416a
[INFO 24-09-03 16:52:17.5442 UTC abstract_model.cc:1362] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 24-09-03 16:52:17.5442 UTC kernel.cc:1061] Use fast generic engine


i:40


[WARNING 24-09-03 16:52:17.9494 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:17.9495 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:17.9495 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:52:18.4032 UTC kernel.cc:1233] Loading model from path /tmp/tmpq9umje6v/model/ with prefix cf80b3511c3745a5
[INFO 24-09-03 16:52:18.4072 UTC kernel.cc:1061] Use fast generic engine


i:41


[WARNING 24-09-03 16:52:18.8103 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:18.8103 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:18.8103 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:52:20.0216 UTC kernel.cc:1233] Loading model from path /tmp/tmpo71g_6jk/model/ with prefix a9a58497b43b413b
[INFO 24-09-03 16:52:20.0395 UTC kernel.cc:1061] Use fast generic engine


i:42


[WARNING 24-09-03 16:52:20.4975 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:20.4975 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:20.4975 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:52:21.0724 UTC kernel.cc:1233] Loading model from path /tmp/tmpt8xddlte/model/ with prefix cc47a0dd791a4b4d
[INFO 24-09-03 16:52:21.0831 UTC kernel.cc:1061] Use fast generic engine


i:43


[WARNING 24-09-03 16:52:21.5071 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:21.5071 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:21.5071 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:52:22.2439 UTC kernel.cc:1233] Loading model from path /tmp/tmp385kk62m/model/ with prefix d9838f019da2478e
[INFO 24-09-03 16:52:22.2619 UTC kernel.cc:1061] Use fast generic engine


i:44


[WARNING 24-09-03 16:52:22.6896 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:22.6896 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:22.6897 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:52:23.2519 UTC kernel.cc:1233] Loading model from path /tmp/tmp18hk4gt3/model/ with prefix c1b14fa8d43a41ff
[INFO 24-09-03 16:52:23.2637 UTC kernel.cc:1061] Use fast generic engine


i:45


[WARNING 24-09-03 16:52:23.8205 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:23.8206 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:23.8206 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:52:24.2756 UTC kernel.cc:1233] Loading model from path /tmp/tmp6x7iptx5/model/ with prefix 2cfdd3a89498424c
[INFO 24-09-03 16:52:24.2790 UTC kernel.cc:1061] Use fast generic engine


i:46


[WARNING 24-09-03 16:52:24.7018 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:24.7019 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:24.7019 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:52:25.4023 UTC kernel.cc:1233] Loading model from path /tmp/tmp7l6em_eb/model/ with prefix 802af3774653416b
[INFO 24-09-03 16:52:25.4157 UTC kernel.cc:1061] Use fast generic engine


i:47


[WARNING 24-09-03 16:52:25.8371 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:25.8371 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:25.8371 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:52:26.4830 UTC kernel.cc:1233] Loading model from path /tmp/tmp_uobrhv6/model/ with prefix 9317a44ff6594d73
[INFO 24-09-03 16:52:26.4949 UTC kernel.cc:1061] Use fast generic engine


i:48


[WARNING 24-09-03 16:52:26.9100 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:26.9100 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:26.9100 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:52:27.3682 UTC kernel.cc:1233] Loading model from path /tmp/tmpe9_cs_rp/model/ with prefix 7c9d5035007b4281
[INFO 24-09-03 16:52:27.3723 UTC kernel.cc:1061] Use fast generic engine


i:49


[WARNING 24-09-03 16:52:27.7928 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:27.7928 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:27.7929 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:52:28.3044 UTC kernel.cc:1233] Loading model from path /tmp/tmphjtkkvqq/model/ with prefix 881a0dff43a5438d
[INFO 24-09-03 16:52:28.3104 UTC abstract_model.cc:1362] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 24-09-03 16:52:28.3105 UTC kernel.cc:1061] Use fast generic engine


i:50


[WARNING 24-09-03 16:52:28.7315 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:28.7315 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:28.7316 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:52:29.3519 UTC kernel.cc:1233] Loading model from path /tmp/tmp2hv_eshh/model/ with prefix 0d97f48650784f86
[INFO 24-09-03 16:52:29.3629 UTC kernel.cc:1061] Use fast generic engine


i:51


[WARNING 24-09-03 16:52:29.7902 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:29.7902 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:29.7902 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:52:30.5085 UTC kernel.cc:1233] Loading model from path /tmp/tmpm70vclmq/model/ with prefix 877fd43c762f40d0
[INFO 24-09-03 16:52:30.5227 UTC kernel.cc:1061] Use fast generic engine


i:52


[WARNING 24-09-03 16:52:30.9429 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:30.9429 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:30.9429 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:52:31.4697 UTC kernel.cc:1233] Loading model from path /tmp/tmp3upc8sg4/model/ with prefix a8b32f2fa5164f04
[INFO 24-09-03 16:52:31.4766 UTC kernel.cc:1061] Use fast generic engine


i:53


[WARNING 24-09-03 16:52:31.8965 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:31.8965 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:31.8965 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:52:32.4168 UTC kernel.cc:1233] Loading model from path /tmp/tmpiz8613p5/model/ with prefix 220b9d3a25914350
[INFO 24-09-03 16:52:32.4235 UTC kernel.cc:1061] Use fast generic engine


i:54


[WARNING 24-09-03 16:52:32.8316 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:32.8316 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:32.8316 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:52:33.2650 UTC kernel.cc:1233] Loading model from path /tmp/tmp_raiyzmq/model/ with prefix 869b795f574f4657
[INFO 24-09-03 16:52:33.2680 UTC kernel.cc:1061] Use fast generic engine


i:55


[WARNING 24-09-03 16:52:33.6705 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:33.6706 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:33.6706 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:52:34.3410 UTC kernel.cc:1233] Loading model from path /tmp/tmpvfdkbb0c/model/ with prefix 874d383e0f2b4e35
[INFO 24-09-03 16:52:34.3543 UTC kernel.cc:1061] Use fast generic engine


i:56


[WARNING 24-09-03 16:52:34.7651 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:34.7652 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:34.7652 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:52:35.3503 UTC kernel.cc:1233] Loading model from path /tmp/tmpnyq4x_wk/model/ with prefix 88bdf00339b842a3
[INFO 24-09-03 16:52:35.3596 UTC quick_scorer_extended.cc:911] The binary was compiled without AVX2 support, but your CPU supports it. Enable it for faster model inference.
[INFO 24-09-03 16:52:35.3603 UTC kernel.cc:1061] Use fast generic engine


i:57


[WARNING 24-09-03 16:52:35.7719 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:35.7719 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:35.7719 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:52:36.2195 UTC kernel.cc:1233] Loading model from path /tmp/tmp3zx5i0pi/model/ with prefix 844a13b669ed4cf0
[INFO 24-09-03 16:52:36.2232 UTC kernel.cc:1061] Use fast generic engine


i:58


[WARNING 24-09-03 16:52:36.6277 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:36.6277 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:36.6277 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:52:37.1151 UTC kernel.cc:1233] Loading model from path /tmp/tmp4lmtth3k/model/ with prefix def6853e566b4547
[INFO 24-09-03 16:52:37.1209 UTC kernel.cc:1061] Use fast generic engine


i:59


[WARNING 24-09-03 16:52:37.5266 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:37.5266 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:37.5266 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:52:38.0801 UTC kernel.cc:1233] Loading model from path /tmp/tmpgywny34b/model/ with prefix 34e566fa4331421a
[INFO 24-09-03 16:52:38.0883 UTC kernel.cc:1061] Use fast generic engine


i:60


[WARNING 24-09-03 16:52:38.4953 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:38.4953 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:38.4953 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:52:39.0626 UTC kernel.cc:1233] Loading model from path /tmp/tmp9_7rfkg9/model/ with prefix ee56e4ede55e405c
[INFO 24-09-03 16:52:39.0713 UTC abstract_model.cc:1362] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 24-09-03 16:52:39.0713 UTC kernel.cc:1061] Use fast generic engine


i:61


[WARNING 24-09-03 16:52:39.4789 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:39.4789 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:39.4790 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:52:39.9726 UTC kernel.cc:1233] Loading model from path /tmp/tmp5bk6fett/model/ with prefix 9b0d025417684e63
[INFO 24-09-03 16:52:39.9771 UTC kernel.cc:1061] Use fast generic engine


i:62


[WARNING 24-09-03 16:52:40.3841 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:40.3841 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:40.3841 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:52:41.0034 UTC kernel.cc:1233] Loading model from path /tmp/tmpvjb6busj/model/ with prefix 9338b78e86ba4fc1
[INFO 24-09-03 16:52:41.0143 UTC kernel.cc:1061] Use fast generic engine


i:63


[WARNING 24-09-03 16:52:41.4264 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:41.4264 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:41.4264 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:52:41.9724 UTC kernel.cc:1233] Loading model from path /tmp/tmp5y7lwvjh/model/ with prefix bd8a727aab1a4972
[INFO 24-09-03 16:52:41.9807 UTC kernel.cc:1061] Use fast generic engine


i:64


[WARNING 24-09-03 16:52:42.3832 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:42.3832 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:42.3833 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:52:42.9021 UTC kernel.cc:1233] Loading model from path /tmp/tmplzc3nva1/model/ with prefix a7aff01aee954bc0
[INFO 24-09-03 16:52:42.9091 UTC kernel.cc:1061] Use fast generic engine


i:65


[WARNING 24-09-03 16:52:43.3160 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:43.3161 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:43.3161 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:52:43.7704 UTC kernel.cc:1233] Loading model from path /tmp/tmp0vf5fumb/model/ with prefix 5e7291c77afd4d11
[INFO 24-09-03 16:52:43.7749 UTC kernel.cc:1061] Use fast generic engine


i:66


[WARNING 24-09-03 16:52:44.1797 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:44.1797 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:44.1797 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:52:44.6710 UTC kernel.cc:1233] Loading model from path /tmp/tmp2touwjtt/model/ with prefix b24a3231e9494f00
[INFO 24-09-03 16:52:44.6768 UTC kernel.cc:1061] Use fast generic engine


i:67


[WARNING 24-09-03 16:52:45.0906 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:45.0906 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:45.0906 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:52:45.8452 UTC kernel.cc:1233] Loading model from path /tmp/tmpkpey_rwt/model/ with prefix a7d175db3b0c405c
[INFO 24-09-03 16:52:45.8624 UTC kernel.cc:1061] Use fast generic engine


i:68


[WARNING 24-09-03 16:52:46.2853 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:46.2853 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:46.2853 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:52:46.9013 UTC kernel.cc:1233] Loading model from path /tmp/tmpkx5xjchw/model/ with prefix 3141363e6e2c40fe
[INFO 24-09-03 16:52:46.9118 UTC kernel.cc:1061] Use fast generic engine


i:69


[WARNING 24-09-03 16:52:47.9654 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:47.9654 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:47.9654 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:52:48.5374 UTC kernel.cc:1233] Loading model from path /tmp/tmpl01u3272/model/ with prefix 3aba52d93dbb4bc8
[INFO 24-09-03 16:52:48.5448 UTC kernel.cc:1061] Use fast generic engine


i:70


[WARNING 24-09-03 16:52:49.0094 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:49.0094 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:49.0095 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:52:49.5800 UTC kernel.cc:1233] Loading model from path /tmp/tmpo9btfsh2/model/ with prefix 7372f262aaac47ae
[INFO 24-09-03 16:52:49.5892 UTC abstract_model.cc:1362] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 24-09-03 16:52:49.5892 UTC kernel.cc:1061] Use fast generic engine


i:71


[WARNING 24-09-03 16:52:50.0503 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:50.0503 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:50.0503 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:52:50.5763 UTC kernel.cc:1233] Loading model from path /tmp/tmphe132bts/model/ with prefix 5826dc6ce615410f
[INFO 24-09-03 16:52:50.5838 UTC kernel.cc:1061] Use fast generic engine


i:72


[WARNING 24-09-03 16:52:51.0062 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:51.0063 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:51.0063 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:52:51.7034 UTC kernel.cc:1233] Loading model from path /tmp/tmppi4jzxu_/model/ with prefix d923e64d7ebf4888
[INFO 24-09-03 16:52:51.7186 UTC kernel.cc:1061] Use fast generic engine


i:73


[WARNING 24-09-03 16:52:52.1391 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:52.1391 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:52.1391 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:52:52.6511 UTC kernel.cc:1233] Loading model from path /tmp/tmpttbcj2za/model/ with prefix 09cc83245bd9478f
[INFO 24-09-03 16:52:52.6576 UTC kernel.cc:1061] Use fast generic engine


i:74


[WARNING 24-09-03 16:52:53.0727 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:53.0727 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:53.0727 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:52:53.6795 UTC kernel.cc:1233] Loading model from path /tmp/tmpyyszjup5/model/ with prefix 751e836ba6f54b1e
[INFO 24-09-03 16:52:53.6901 UTC kernel.cc:1061] Use fast generic engine


i:75


[WARNING 24-09-03 16:52:54.1046 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:54.1047 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:54.1047 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:52:54.6461 UTC kernel.cc:1233] Loading model from path /tmp/tmpvfwre8pb/model/ with prefix 0a56c5beab854dce
[INFO 24-09-03 16:52:54.6536 UTC kernel.cc:1061] Use fast generic engine


i:76


[WARNING 24-09-03 16:52:55.0839 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:55.0840 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:55.0840 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:52:55.5934 UTC kernel.cc:1233] Loading model from path /tmp/tmp4uvwaza5/model/ with prefix 4aedef381de24d76
[INFO 24-09-03 16:52:55.5973 UTC kernel.cc:1061] Use fast generic engine


i:77


[WARNING 24-09-03 16:52:56.0310 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:56.0311 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:56.0311 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:52:56.5061 UTC kernel.cc:1233] Loading model from path /tmp/tmp4y4g0p94/model/ with prefix 091188c9aaef4a8f
[INFO 24-09-03 16:52:56.5104 UTC kernel.cc:1061] Use fast generic engine


i:78


[WARNING 24-09-03 16:52:56.9191 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:56.9191 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:56.9191 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:52:57.4192 UTC kernel.cc:1233] Loading model from path /tmp/tmpm6fi0b11/model/ with prefix a9692ddb35654840
[INFO 24-09-03 16:52:57.4251 UTC kernel.cc:1061] Use fast generic engine


i:79


[WARNING 24-09-03 16:52:57.8342 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:57.8343 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:57.8343 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:52:58.3383 UTC kernel.cc:1233] Loading model from path /tmp/tmpj617o1gf/model/ with prefix d948e8216e794be8
[INFO 24-09-03 16:52:58.3446 UTC kernel.cc:1061] Use fast generic engine


i:80


[WARNING 24-09-03 16:52:58.7436 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:58.7436 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:58.7436 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:52:59.3009 UTC kernel.cc:1233] Loading model from path /tmp/tmpbljm_a4p/model/ with prefix 8eabffabf75b4967
[INFO 24-09-03 16:52:59.3098 UTC kernel.cc:1061] Use fast generic engine


i:81


[WARNING 24-09-03 16:52:59.7124 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:59.7124 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:52:59.7124 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:53:00.3555 UTC kernel.cc:1233] Loading model from path /tmp/tmpiuvmh95k/model/ with prefix fcd3a0e342b2498f
[INFO 24-09-03 16:53:00.3655 UTC abstract_model.cc:1362] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 24-09-03 16:53:00.3655 UTC kernel.cc:1061] Use fast generic engine


i:82


[WARNING 24-09-03 16:53:00.7686 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:53:00.7686 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:53:00.7686 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:53:01.3247 UTC kernel.cc:1233] Loading model from path /tmp/tmpltm_f58w/model/ with prefix 4e3292374c854793
[INFO 24-09-03 16:53:01.3334 UTC kernel.cc:1061] Use fast generic engine


i:83


[WARNING 24-09-03 16:53:01.7397 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:53:01.7398 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:53:01.7398 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:53:02.2799 UTC kernel.cc:1233] Loading model from path /tmp/tmpilnfi07o/model/ with prefix 7616f276fa484462
[INFO 24-09-03 16:53:02.2877 UTC kernel.cc:1061] Use fast generic engine


i:84


[WARNING 24-09-03 16:53:02.6955 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:53:02.6956 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:53:02.6956 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:53:03.3819 UTC kernel.cc:1233] Loading model from path /tmp/tmpmmmifqyi/model/ with prefix 66cc3b6eb5c342c0
[INFO 24-09-03 16:53:03.3962 UTC kernel.cc:1061] Use fast generic engine


i:85


[WARNING 24-09-03 16:53:03.8163 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:53:03.8163 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:53:03.8164 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:53:04.3159 UTC kernel.cc:1233] Loading model from path /tmp/tmpip7cmet3/model/ with prefix ca60953fc1e347ff
[INFO 24-09-03 16:53:04.3220 UTC kernel.cc:1061] Use fast generic engine


i:86


[WARNING 24-09-03 16:53:04.7437 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:53:04.7438 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:53:04.7438 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:53:05.4412 UTC kernel.cc:1233] Loading model from path /tmp/tmpkngq0ijx/model/ with prefix b0d19441deb34604
[INFO 24-09-03 16:53:05.4549 UTC quick_scorer_extended.cc:911] The binary was compiled without AVX2 support, but your CPU supports it. Enable it for faster model inference.
[INFO 24-09-03 16:53:05.4558 UTC kernel.cc:1061] Use fast generic engine


i:87


[WARNING 24-09-03 16:53:05.8823 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:53:05.8823 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:53:05.8823 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:53:06.6254 UTC kernel.cc:1233] Loading model from path /tmp/tmpkgn0r9mf/model/ with prefix 37097b05a64f4313
[INFO 24-09-03 16:53:06.6414 UTC kernel.cc:1061] Use fast generic engine


i:88


[WARNING 24-09-03 16:53:07.0573 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:53:07.0573 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:53:07.0573 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:53:07.6781 UTC kernel.cc:1233] Loading model from path /tmp/tmp8fzeubmx/model/ with prefix acc7d08b98bb4799
[INFO 24-09-03 16:53:07.6883 UTC kernel.cc:1061] Use fast generic engine


i:89


[WARNING 24-09-03 16:53:08.1091 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:53:08.1091 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:53:08.1091 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:53:08.5845 UTC kernel.cc:1233] Loading model from path /tmp/tmpqgje053b/model/ with prefix 26a16ffd50d74662
[INFO 24-09-03 16:53:08.5889 UTC kernel.cc:1061] Use fast generic engine


i:90


[WARNING 24-09-03 16:53:08.9943 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:53:08.9943 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:53:08.9943 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:53:09.5769 UTC kernel.cc:1233] Loading model from path /tmp/tmpyn7ewgw5/model/ with prefix 9db4f865de42498b
[INFO 24-09-03 16:53:09.5862 UTC kernel.cc:1061] Use fast generic engine


i:91


[WARNING 24-09-03 16:53:10.0186 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:53:10.0186 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:53:10.0186 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:53:10.5304 UTC kernel.cc:1233] Loading model from path /tmp/tmpngqu309m/model/ with prefix 955fa2c266804d95
[INFO 24-09-03 16:53:10.5367 UTC abstract_model.cc:1362] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 24-09-03 16:53:10.5368 UTC kernel.cc:1061] Use fast generic engine


i:92


[WARNING 24-09-03 16:53:10.9442 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:53:10.9442 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:53:10.9443 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:53:11.6852 UTC kernel.cc:1233] Loading model from path /tmp/tmpf21t1rxg/model/ with prefix a809b1fea6664f13
[INFO 24-09-03 16:53:11.7013 UTC kernel.cc:1061] Use fast generic engine


i:93


[WARNING 24-09-03 16:53:12.1119 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:53:12.1119 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:53:12.1119 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:53:12.6870 UTC kernel.cc:1233] Loading model from path /tmp/tmpyj_az573/model/ with prefix 33e486d99a7f4aa6
[INFO 24-09-03 16:53:12.6959 UTC kernel.cc:1061] Use fast generic engine


i:94


[WARNING 24-09-03 16:53:13.1012 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:53:13.1012 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:53:13.1012 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:53:13.6003 UTC kernel.cc:1233] Loading model from path /tmp/tmpyfk_auty/model/ with prefix aab02913afcf43b4
[INFO 24-09-03 16:53:13.6062 UTC kernel.cc:1061] Use fast generic engine


i:95


[WARNING 24-09-03 16:53:14.6742 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:53:14.6743 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:53:14.6743 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:53:15.2591 UTC kernel.cc:1233] Loading model from path /tmp/tmpnpv9xo3a/model/ with prefix 2eedeb60e0b94025
[INFO 24-09-03 16:53:15.2683 UTC kernel.cc:1061] Use fast generic engine


i:96


[WARNING 24-09-03 16:53:15.7242 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:53:15.7243 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:53:15.7243 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:53:16.3125 UTC kernel.cc:1233] Loading model from path /tmp/tmpccnpxf8k/model/ with prefix e31eea11140e40bf
[INFO 24-09-03 16:53:16.3231 UTC kernel.cc:1061] Use fast generic engine


i:97


[WARNING 24-09-03 16:53:16.7533 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:53:16.7533 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:53:16.7534 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:53:17.2506 UTC kernel.cc:1233] Loading model from path /tmp/tmptn6sucgw/model/ with prefix ae6bed9492b34781
[INFO 24-09-03 16:53:17.2557 UTC kernel.cc:1061] Use fast generic engine


i:98


[WARNING 24-09-03 16:53:17.6776 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:53:17.6776 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:53:17.6777 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:53:18.2631 UTC kernel.cc:1233] Loading model from path /tmp/tmpe53szpvs/model/ with prefix f69869813644442e
[INFO 24-09-03 16:53:18.2707 UTC kernel.cc:1061] Use fast generic engine


i:99


[WARNING 24-09-03 16:53:18.6894 UTC gradient_boosted_trees.cc:1840] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:53:18.6894 UTC gradient_boosted_trees.cc:1851] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-09-03 16:53:18.6894 UTC gradient_boosted_trees.cc:1865] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-09-03 16:53:19.3659 UTC kernel.cc:1233] Loading model from path /tmp/tmp24esrnnt/model/ with prefix 2e654eb18049475d
[INFO 24-09-03 16:53:19.3795 UTC kernel.cc:1061] Use fast generic engine


Submission exported to /kaggle/working/submission.csv
